javatests/arcs/core/data/proto/SchemaProtoDecoderTest.kt

```diff
@@ -22,6 +22,10 @@ fun decodeSchemaProtoText(protoText: String): Schema {
 
 @RunWith(JUnit4::class)
 class SchemaProtoDecoderTest {
+
```

Comment:
Can you move these comments to inside the tests where you declare the text protos?

Reply:
Inside the test methods?

Reply:
I meant something like this, instead of annotating the test function:
```suggestion
    @Test
    fun decodesNamesInSchemaProto() {
         // schema Thing, Object
         val schemaProtoText = """
           names:  "Thing"
           names:  "Object"
         """.trimIndent()
         ...
    }
```